src/parser/parser.h

```diff
@@ -527,11 +527,15 @@ class CVC4_PUBLIC Parser {
                  bool levelZero = false, bool doOverload = false);
 
   /** Create a new type definition. */
```

Comment:
Can you add documentation for the arguments (especially the added one).

Reply:
Good point, done.